package.json

```
@@ -746,6 +746,37 @@
           ],
           "default": false
         },
+        "objectscript.export.map": {
+          "markdownDescription": "Map file names before export with regexp pattern as a key, and replacement as a value\n (e.g. ```JSON\n {\n  \"%(.*)\": \"_$1\"\n}\n``` for % class or routines replaced to be with underscore instead).",
```

```suggestion
          "markdownDescription": "Map file names before export, with regexp pattern as a key and replacement as a value (e.g. `{  \"%(.*)\": \"_$1\" }` to make % classes or routines use underscore prefix instead).",
```
Improve wording, and remove Markdown elements not supported in this property.